constructor/main.py

```diff
@@ -109,6 +109,12 @@ def main_build(dir_path, output_dir='.', platform=cc_platform,
         if info.get(key):  # only join if there's a truthy value set
             info[key] = abspath(join(dir_path, info[key]))
 
+    # Normalize name and set default value
+    if info.get("windows_signing_tool"):
+        info["windows_signing_tool"] = info["windows_signing_tool"].lower().replace(".exe", "")
+    elif info.get("signing_certificate"):
+        info["windows_signing_tool"] = "signtool"
```

Comment:
We should check if we are overriding something here, and warn appropriately.

Reply:
That check is done in line 113 - if `windows_signing_tool` is set in any way, we won't get to `elif`.